lms/djangoapps/discussion_api/tests/test_serializers.py

```diff
@@ -184,37 +184,18 @@ def test_basic(self):
             "unread_comments_count": 3,
             "read": False,
             "endorsed": False,
-            "response_count": None,
+            "resp_total": 0,
```

Comment:
1. Why doesn't this use `make_minimal_cs_thread`?
2. Does this represent the actual fix?